src/python/aqueduct_executor/operators/utils/utils.py

```diff
@@ -267,7 +267,10 @@ def write_delete_saved_objects_results(
 ) -> None:
     results_str = json.dumps(
         {
-            integration: [result.__dict__ for result in results[integration]]
+            integration: [
+                {"name": result.name, "exec_state": json.loads(result.exec_state.json())}
```

Comment:
[class].json() gives me the json as a string which gets escaped when I do `json.dumps`. However, I cannot serialize it directly with `json.dumps` so I serialize with `.json()` then `json.loads()` to get the format that is serializable by `json.dumps`.